README.md

```diff
@@ -86,6 +86,37 @@ It is manadatory to load an authorization JSON file, if no authorization file is
 
 The keycloak event-emitter module sends all events to the bridge's event endpoint. The event emitter use HTTP with flatbuffers.
 
+### Monitoring of keycloak-bridge
+
+An endpoint allows to get a status of the Bridge and its components health.
+URL: ```http://{bridge.host}:{bridge.port}/health/check```
+
+Status example:
+```
+{
+ "Audit R/W": {
+  "state": "UP",
+  "connection": "established"
+ },
+ "Audit RO": {
+  "state": "DOWN",
+  "message": "commands out of sync. You can't run this command now"
```

Comment:
In what case we would have this message?

Reply:
It is the message sent by the MariaDB driver when you're trying to execute a SQL query while the database has been shut down.

Reply:
Thanks